CMakeLists.txt

```
@@ -701,8 +701,12 @@ if(SDLMIXER_GME)
         set(BUILD_SHARED_LIBS "${SDLMIXER_GME_SHARED}")
         set(ENABLE_UBSAN OFF)
         set(BUILD_FRAMEWORK OFF)
-        set(GME_ZLIB OFF)
-        set(GME_UNRAR OFF)
+        if(NOT DEFINED GME_ZLIB)
+            set(GME_ZLIB OFF)
+        endif()
+        if(NOT DEFINED GME_UNRAR)
+            set(GME_UNRAR OFF)
+        endif()
```

Would this work for you too?
```suggestion
        set(GME_ZLIB "OFF" CACHE BOOL "Build GME with zlib support")
        set(GME_UNRAR "OFF" CACHE BOOL "Build GME with unrar support")
```

---

It doesn't work for me. I am setting GME_ZLIB from the project CMakeLists.txt before including SDL_mixer.

---

You need to do the following in your cmake script:
```cmake
set(GME_ZLIB ON CACHE BOOL "GME with zlib" FORCE)
set(GME_UNRAR ON CACHE BOOL "GME with unrar" FORCE)
```

---

Now it works, but as user just writing:
```cmake
set(GME_ZLIB ON)
```
seems like more intuitive way for me.

---

This one has a better behavior, I think.
```suggestion
        option(GME_ZLIB "GME with zlib" OFF)
        option(GME_UNRAR "GME with unrar" OFF)
```

---

Yes, this seems like a better way. Also libgme CMakeLists no longer has GME_UNRAR option.